doc/source/examples/antenna_toolkit_example.py

```diff
@@ -0,0 +1,225 @@
+# # Antenna toolkit example
+#
+# This example demonstrates how to use the ``ToolkitBackend`` class.
+# It initiates AEDT through PyAEDT, sets up an empty HFSS design, and proceeds to create the antenna.
+
+
+# ## Perform required imports
+
+import tempfile
+
+import pyaedt
+from pyaedt.modules.solutions import FfdSolutionData
+
+from ansys.aedt.toolkits.antenna.backend.api import ToolkitBackend
+from ansys.aedt.toolkits.antenna.backend.models import properties
+
+# ##  Set AEDT version
+#
+# Set AEDT version.
```

Comment:
Is there a reason that the heading here and in the next few lines is repeated twice?